test/acceptance/README.md

```diff
@@ -1,10 +1,212 @@
+
+To run acceptance smoke tests with local instance of SBO:
+
+```bash
+make test-acceptance-smoke
+.
+.
+.
+```
+
+This command will prepare the environment similarly to the previous one and:
+* Builds and start a local instance of the operator
+* Executes the `smoke` sub-set of acceptance tests [features and scenarios](https://github.com/redhat-developer/service-binding-operator/tree/master/test/acceptance/features) - marked with `@smoke` tag.
+
+### Run acceptance tests with remote instance of SBO
+
+It is possible to run the acceptance tests with SBO already running, that is without building and starting the local instance and without installing the dependent resources (such as CRD, role, role-binding, service-account, etc.) required when the local instance is used. That is particulary useful for testing SBO installed on a remote cluster or running the acceptance tests with various versions of the SBO.
+
+For that there is `TEST_ACCEPTANCE_START_SBO` environment variable that needs to be set to `remote`.
+
+The remote cluster where the SBO should be running is determined by the `KUBECONFIG`.
+
+To run acceptance tests with remote instance of SBO:
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote make test-acceptance
+```
+
+Compared with the defaults this command only:
+* Deletes the old namespace from any previous run,
+* Creates a new namespace with generated name like `test-namespace-xxxxxxxx`,
+* Executes the acceptance tests.
+
+The `remote` option requires to specify the namespace where the operator is running by setting the `SBO_NAMESPACE` env variable.
+
+To run acceptance tests with remote instance of SBO running in a given namespace (e.g. `sbo`):
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote SBO_NAMESPACE=sbo make test-acceptance
+```
+
+
+### Run a sub-set of features or scenarios
+
+It is possible to run a sub-set of features or even a single scenario (for example, when you are working on a new scenario and you need to run it over and over). The way how to do it is by "marking" that feature or scenario by a "tag" - directly in the particular `*.feature` file - in a form of `@<tag>` (e.g. `@wip`). Something like the following:
+
+```gherkin
+Feature: ...
+
+   @wip
+   Scenario: Bind service 1 to app 1
+   ...
+
+   Scenario: Bind service 1 to app 2
+   ...
+```
+
+Now, when you want to execute just that scenario out of the whole suite, use the `TEST_ACCEPTANCE_TAGS` environmental variable to set the tag.
+
+To run single scenario tagged with `@wip`:
+
+```bash
+TEST_ACCEPTANCE_TAGS=@wip make test-acceptance
+.
+.
+.
+
+```
+
+Only the scenario or feature tagged with the particular tag (`@wip` in case of our example) will be executed, and all of the others will be simply skipped.
+
+```bash
+.
+.
+.
+
+1 feature passed, 0 failed, 3 skipped
+1 scenario passed, 0 failed, 16 skipped
+10 steps passed, 0 failed, 179 skipped, 0 undefined
+Took 0m49.231s
+```
+
+That's all Folks! ...well, at least for now.
```

Comment:
Should we mention somewhere `test-acceptance-report` rule and its usage? Or how to visualize the reports?

Reply:
Done